src/components/ContactMoreInfo.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <div>
+    <hr class="line" />
+    <div class="info">
+      <table>
```

Comment:
I would like to avoid using html tables as much as possible, can we use flex for things like this?